.github/CODEOWNERS

```
@@ -1 +1,4 @@
 * @signalfx/gdi-js-maintainers @signalfx/gdi-js-web-approvers
+*.md @signalfx/docs
+*.rst @signalfx/docs
+docs/* @signalfx/docs
```

[Wouldn't match nested files](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#example-of-a-codeowners-file)

```suggestion
docs/ @signalfx/docs
```